src/styles/blocks/card.css

```diff
@@ -0,0 +1,119 @@
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+
+  max-width: 200px;
+  padding: 0 16px 16px;
+
+  background-color: #fff;
```

Comment:
we can use var for background-color

Reply:
> we can use var for background-color

we use it one time, so think dont need to make variable, probably